e2e/components/OverflowMenu/OverflowMenu-test.avt.e2e.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('OverflowMenu', () => {
+  test('accessibility-checker @avt', async ({ page }) => {
```

Comment:
Minor thing, but you can put the @avt flag in the describe block and then you don't have to add it to every test name. As this file grows could be helpful to ensure everything in the block will get picked up by the test runner.

```suggestion
test.describe('OverflowMenu @avt', () => {
  test('accessibility-checker', async ({ page }) => {
```

Reply:
Cool! Fixed!

Also just found a way to filter down descendant elements
```
 await expect(
      page.locator('button').filter({ hasText: 'Stop app' })
    ).toBeFocused();
```